core/src/main/java/org/stellar/anchor/config/CustodySecretConfig.java

```diff
@@ -1,9 +1,13 @@
 package org.stellar.anchor.config;
 
+import javax.crypto.SecretKey;
+
 public interface CustodySecretConfig {
   String getFireblocksApiKey();
 
   String getFireblocksSecretKey();
 
   String getCustodyAuthSecret();
+
+  SecretKey getCustodyAuthSecretKey();
```

Comment:
The `SecretKey` does not belong to the config class. We should move it out to avoid confusion.

Reply:
The reason I added it here is to validate it the same time it's created. Otherwise it can be easily missed. Plus, it's easier to work with it when it's stored in the config. Do you think we should remove it? Where should we keep validation?